tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/OrderItemServiceTests.cs

```diff
@@ -111,14 +111,14 @@ public static void Constructor_NullAccessor_ThrowsArgumentNullException()
 
         [Test]
         [InMemoryDbAutoData]
-        public static async Task GetOrderItems_ReturnsEmpty(
+        public static async Task GetOrderItems_ReturnsNull(
```

Comment:
This and other test names in the file do not give a full description of what's being tested so its easier to understand what's going on.
Suggest a format of <MethodName>_<InputOrCondition>_<Result> 
e.g. this title could be `GetOrderItems_InvalidCallOffId_ReturnsNull`.

Reply:
With PR [424](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering/pull/424/files)
Renamed unit test method